### PR TITLE
LPS-130877 Add test to untracked instances

### PR DESCRIPTION
**Pull request by seltong**
Issue: [LPS-130877](https://issues.liferay.com/browse/LPS-130877)

### CODE REVIEW

**Comment by seltong** [comment]
ci:test:sf

**Comment by jotascript** [comment]
ci:test:sf

**Review by jotascript — APPROVED** [review]

**Review by ramonmsp — CHANGES_REQUESTED** [review]
@seltong these commits by @naraandrade are part of this fix? If not, can you, please, remove them?

Ty :D

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/instance-list-page/modal/instance-details/InstanceDetailsModalBody.es.js [hunk 1]
```diff
@@ -33,10 +33,26 @@ function Body({
 	slaResults = [],
 	taskNames = [],
 }) {
-	const SLAs = {open: [], resolved: []};
+	const SLAs = {notStarted: [], open: [], resolved: []};
```
Comment (ramonmsp) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/instance-list-page/modal/instance-details/InstanceDetailsModalBody.es.js:36]: Hey @seltong, could you order it in alphabetical?

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/instance-list-page/modal/instance-details/InstanceDetailsModalBody.es.js [hunk 2]
```diff
@@ -33,10 +33,26 @@ function Body({
...
 
 	slaResults.forEach((result) => {
-		SLAs[result.status === 'STOPPED' ? 'resolved' : 'open'].push(result);
+		let slaGroup = '';
+
+		switch (result.status) {
+			case 'STOPPED': {
```
Comment (ramonmsp) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/instance-list-page/modal/instance-details/InstanceDetailsModalBody.es.js:42]: @seltong same here.
We can put 'NEW' before 'STOPPED'

**Review by ramonmsp — CHANGES_REQUESTED** [review]

**Comment by ramonmsp** [comment]
ci:test:sf

**Comment by ramonmsp** [comment]
ci:test:relevant

**Review by ramonmsp — APPROVED** [review]

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-impl/src/main/java/com/liferay/portal/workflow/metrics/rest/internal/dto/v1_0/util/SLAResultUtil.java [hunk 3]
```diff
@@ -97,6 +97,9 @@ else if (Objects.equals(
 
 			return SLAResult.Status.PAUSED;
 		}
+		else if (Objects.equals(status, WorkflowMetricsSLAStatus.NEW.name())) {
```
Comment (rafaprax) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-impl/src/main/java/com/liferay/portal/workflow/metrics/rest/internal/dto/v1_0/util/SLAResultUtil.java:100]: @seltong move this if statement to second position, please. We must use the alphabetical order here as well. :)

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/instance-list-page/modal/instance-details/InstanceDetailsModalBody.es.js (hunk 2 above)
Comment (rafaprax) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/instance-list-page/modal/instance-details/InstanceDetailsModalBody.es.js:42]: @seltong the case statement must be in alphabetical order as well :). Please, fix it.

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by rafaprax** [comment]
Good Job @seltong. Just small changes required. 👏 👏

**Comment by seltong** [comment]
ci:test:sf

**Comment by ramonmsp** [comment]
ci:forward

**Pull request closed without merging**